src/layouts/SetupContainer.tsx

```diff
@@ -12,12 +14,21 @@ export default function SetupContainer({
 }: PropsWithChildren<Props>) {
   return (
     <main className="flex min-h-screen h-full w-screen flex-col items-center justify-center bg-primary-900 transition-colors text-white">
-      <div className="fixed right-16 top-4 flex h-8 w-48 items-center justify-around">
+      <div className="fixed md:right-16 top-16 md:top-8 flex h-8 w-48 md:w-96 md:flex-row flex-col-reverse gap-6 md:gap-4 justify-center items-center">
+        <a href="https://docs.raspiblitz.org/" target="_blank" rel="noreferrer">
```

Comment:
Could this do the job?: 
https://nextui.org/docs/components/link#polymorphic-component

<img width="722" alt="image" src="https://github.com/user-attachments/assets/b102eed1-d9f0-4dd4-bf71-3a907ae39f6f">

Reply:
Wondering if this could be something else then primary to make sure what the main button on this view?  
Maybe _bordered_ or _ghost_ could work?: https://nextui.org/docs/components/button#variants